### DISCUSSION TITLE
Discontinuation

### DISCUSSION THREAD

**Discussion by ludo550** [Shelex/cypress-allure-plugin#226] [General]
Hi @Shelex , Good sir,

I want to use Allure reporter for Cypress but was told you are discontinuing it's maintenance. Is this accurate information?

**Comment by Shelex** [comment]
Hi @ludo550 

The point is that "discontinuing it's maintenance" is quite loud, as in that case I would just mark repo as archived. From my point of view it already serves the purpose it was built for. I just do not have free resources to implement new features or address issues with specific cypress versions due to the russian war against Ukraine where I currently live.

In https://github.com/Shelex/cypress-allure-plugin/issues/220 I mentioned that there may be a lot of reasons to not bring allure on your project (as well as cypress) and that is for you to decide. Any other recommendation would be my subjective opinion.

**Reply by ludo550** [reply]
I hear you @Shelex , Thank you for explaining! And great job on this plugin. Hope everything in Ukraine gets better
